lib/perl/Genome/Model/Tools/Varscan.pm

```diff
@@ -66,14 +89,20 @@ my %VARSCAN_VERSIONS = (
 );
 
 sub java_command_line {
-    my $self = shift;
+    my $self             = shift;
     my $parameter_string = shift;
 
-    my $path = $self->path_for_version($self->version);
-    my $headers = $self->no_headers ? "--no-headers 1" : "";
-    my $command_line = 'java -jar ' . $path . ' ' . $parameter_string . ' '.$headers;
+    my @java_cmd = ( $self->java_interpreter );
+
+    my $path = $self->path_for_version( $self->version );
+    if ( defined $self->max_memory ) {
+        push @java_cmd, sprintf( "-Xmx%dg", $self->max_memory );
+    }
+    push @java_cmd, "-Djava.io.tmpdir=" . $self->tmp_dir, '-jar' => $path;
```

Comment:
`tmp_dir` is currently not being set anywhere.  Perhaps [this fallback from G/M/T/Gatk/Base.pm](https://github.com/genome/genome/blob/5bf928f396ba5921abc6aaf715ccf43353fdb2eb/lib/perl/Genome/Model/Tools/Gatk/Base.pm#L95-L98) would be useful?